blog/2025-03-13.md

```diff
@@ -0,0 +1,45 @@
+---
+title: '📢 Milestone 0.4 has been reached'
+authors: mayya
+tags: [tanager, release]
+---
+
+## What's Changed
+
+The fourth milestone on our journey to a production-ready release of the Data
+Management Service (DMS) introduces Configuration Management Service backed by
+configuration data store, namespace-based authorization, and Keycloak
+integration as the identity provider.
+
+<!-- truncate -->
+
+## Key Updates
+
+1. Introduces [DMS Configuration
+   Service](https://github.com/Ed-Fi-Alliance-OSS/Project-Tanager/tree/main/docs/CS),
+   designed to be compatible with Admin API 2.2
+2. Implements [OAuth
+   support](https://github.com/Ed-Fi-Alliance-OSS/Project-Tanager/blob/main/docs/AUTH.md)
+   via KeyCloak.
+3. Enforces token-based authorization for all Descriptor API and Resource API
+   endpoints in DMS.
+4. Introduced namespace-based authorization for Descriptors and namespace-scoped
+   resources (e.g., Assessments).
+
+Additionally, this milestone included API client SDK generation testing for
+compatibility with ODS API SDK, a basic eTag implementation, and expanded
+end-to-end test cases.
+
+📌[View Full
+Changelog](https://github.com/Ed-Fi-Alliance-OSS/Data-Management-Service/compare/0.3.0...0.4.0)
+
+## Next
+
+The upcoming 0.5.0 milestone will add claimset-based authorization metadata
+served through the DMS Configuration Service, relationship-based authorization
+via student enrollment and staff employment, EPDM extension delivery and support
+for custom model extensions.
+
+For more detail about what comes next, see [Proposed Features and
+Milestones](https://github.com/orgs/Ed-Fi-Alliance-OSS/projects/1/views/2) in
```

Comment:
I was a little confused at first, because this goes to the Roadmap board not the Proposed Features board. But the Roadmap board is the right place to go.